packages/@aws-cdk/aws-apigatewayv2/lib/http/api.ts

```diff
@@ -205,15 +219,20 @@ export interface AddRoutesOptions extends BatchHttpRouteOptions {
 
   /**
    * Authorizer to be associated to these routes.
-   * @default - No authorizer
+   *
+   * Set to 'NONE' to remove the default authorizer for the api
+   *
+   * @default - No authorizer once no default one set on the gateway
    */
-  readonly authorizer?: IHttpRouteAuthorizer;
+  readonly authorizer?: IHttpRouteAuthorizer | 'NONE';
 
   /**
    * The list of OIDC scopes to include in the authorization.
    *
-   * These scopes will be merged with the scopes from the attached authorizer
-   * @default - no additional authorization scopes
+   * These scopes will be merged with the default authorization scopes on the gateway.
```

Comment:
I'm curious why you chose to merge rather than override. If a customer wants most routes to use a standard set of scopes but wants to change this for a specific route, overrides would be better.

As an example, all routes should have 'execute' permissions but a specific 'administrator' API should only have 'write' permissions.

Reply:
Ahh okay. Not sure why i thought that 🙈 . Will make the change